package.json

```diff
@@ -100,5 +97,25 @@
     "ts-loader": "^9.5.1",
     "typescript": "^5.0.4",
     "webpack": "^5.90.3"
+  },
+  "packageManager": "pnpm@8.15.6",
+  "engines": {
+    "node": "^20.0.0",
+    "pnpm": "^8.15.6"
+  },
+  "volta": {
+    "node": "20.12.2",
+    "pnpm": "8.15.6"
+  },
```

Comment:
It is good practice to define these fields to make it clear which version of tooling is supported.